pyproject.toml

```diff
@@ -27,6 +27,7 @@ classifiers = [
 dependencies = [
     "docstring_parser>=0.16",  # Pin for description addition
     "httpx",
+    "numpy",
```

Comment:
Any chance we can avoid adding deps? Sorry I missed this in the original review

We should only add deps if we actually need them